Changelog.md

```diff
@@ -2,6 +2,8 @@
 
 ## 2.3.0 - UNRELEASED
 - Moving @types dependencies into devdependencies
+- Port NO_PROXY and ALL_PROXY support.
+- Port username/password support in proxy url string.
```

Comment:
For ms-rest-js users "porting" wouldnt make sense, they would not have the context. 

```suggestion
- Add NO_PROXY and ALL_PROXY support.
- Add username/password support in proxy url string.
```